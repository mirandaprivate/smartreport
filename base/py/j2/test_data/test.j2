NAME : {{data.name}}
AGE: {{data.age}}
SAY: {{data.say}}

{# Test filters #}
{{ 'http://example.com/foo/bar?k1=v1' | urlsplit('path') }}
{{ 'http://example.com/foo/bar?k1=v1' | urlsplit('scheme') }}
{{ 'http://example.com/foo/bar?k1=v1' | urlsplit('query') }}
