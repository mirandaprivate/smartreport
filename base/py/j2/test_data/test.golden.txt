NAME : Foo
AGE: 12
SAY: Hello world!

/foo/bar
http
k1=v1
